controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/VcmrMaintainer.java

```diff
@@ -357,4 +361,13 @@ protected ZonedDateTime getRetirementStartTime(ZonedDateTime plannedStartTime) {
         return time;
     }
 
+    private void updateMetrics() {
+        curator.readChangeRequests()
+                .stream()
+                .collect(Collectors.groupingBy(VespaChangeRequest::getStatus))
+                .forEach((status, cmrs) ->
+                        metric.set(TRACKED_CMRS_METRIC, cmrs.size(), metric.createContext(Map.of("status", status.name())))
```

Comment:
Might be better to explicitly iterate over statuses, so we properly report 0 instead of no data